config/owasp/suppressions.xml

```diff
@@ -11,4 +11,11 @@
   <suppress>
     <cve>CVE-2024-38820</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+    file name: jose4j-0.9.3.jar
+    ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.bitbucket\.b_c/jose4j@.*$</packageUrl>
+    <cve>CVE-2023-51775</cve>
+  </suppress>
```

Comment:
This CVE doesn't need to be supressed and can be fixed by either upgrading jose4j or the gov notify client